src/components/Garden/Agreement/Agreement.js

```diff
@@ -8,14 +8,14 @@ import LayoutGutter from '../Layout/LayoutGutter'
 import LayoutLimiter from '../Layout/LayoutLimiter'
 import LayoutBox from '../Layout/LayoutBox'
 import LayoutColumns from '../Layout/LayoutColumns'
-import MultiModal from '../MultiModal/MultiModal'
+import MultiModal from '@components/MultiModal/MultiModal'
 import SignAgreementScreens from '../ModalFlows/SignAgreementScreens/SignAgreementScreens'
-import { useAgreement } from '../../hooks/useAgreement'
-import { useWallet } from '../../providers/Wallet'
+import { useAgreement } from '@hooks/useAgreement'
+import { useWallet } from '@providers/Wallet'
 
-import Loader from '../Loader'
+import Loader from '../../Loader'
 
-import warningSvg from '../../assets/warning.svg'
+import warningSvg from '../../../assets/warning.svg'
```

Comment:
Wondering if we should move this asset to `./assets`  since it's only being used by the agreement